roles/tunnel-server/tasks/certificates.yml

```diff
@@ -75,6 +75,17 @@
   notify:
     - Restart Nginx
 
+- name: Re-generate users.json
+  when: app_domain_certs_exist|d(False) == True
+  template:
+    src: users.json.j2
+    dest: "{{ app_webroot }}/users.json"
+    owner: "{{ nginx_user }}"
+    group: "{{ nginx_user }}"
+    mode: 0755
+  notify:
+    - Restart Nginx
```

Comment:
Ansible step to generate that `users.json` file from the template below

Reply:
interesting! 🥇